packages/accounts-express/create_auth_middleware.js

```
@@ -0,0 +1,78 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts, _CurrentEndpointInvocation } from 'meteor/accounts-base';
+
+export function createWebAppAuthMiddleware({ hashLoginTokenFn, required = false }) {
+  return async function meteorWebAppAuthMiddleware(req, res, next) {
+    const authHeader = req.headers.authorization;
+    const cookies = req.headers.cookie;
+    let token;
+
+    // Try to get token from Authorization header
+    if (authHeader?.startsWith("Bearer ")) {
```

ESLint

[error] Replace `"Bearer·"` with `'Bearer·'`

(prettier/prettier)